scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -221,14 +229,26 @@ def net_start(args):
     netpath = args.netpath
     netname = xml_get_value(args.netpath, "name")
     print("defining network from {0}".format(netname))
-    networks = conn.listNetworks()
-    if netname not in networks:
-        print("defining network from {0}".format(netpath))
-        xml = readfile(netpath)
-        conn.networkDefineXML(xml)
-    print("starting {0} network".format(netname))
-    dom = conn.networkLookupByName(netname)
-    dom.create()
+
+    # Get the names of active and inactive network domains.
+    try:
+        networks = [network.name() for network in conn.listAllNetworks()]
+        # If network domain exists
+        if netname in networks:
+            net_dom = conn.networkLookupByName(netname)
+            # Check if active, if not set autostart and activate the network
+            if not net_dom.isActive():
+                print("starting {0} network".format(netname))
+                net_dom.setAutostart(1)
```

Comment:
Thanks, I will update this. But IMO, its a painful process to re-activate the networks everytime I do a reboot on my workstation and want to re-use the cloud.